dts/bindings/sensor/ti,tmp114.yaml

```diff
@@ -6,3 +6,20 @@ description: Texas Instruments TMP114 temperature sensor
 compatible: "ti,tmp114"
 
 include: [sensor-device.yaml, i2c-device.yaml]
+
+properties:
+  odr:
+    type: int
+    default: 4
+    description: |
+      Specify the default temperature output data rate in milliseconds (ms).
+      Default is power-up configuration (250ms).
+    enum:
+      - 0    # TMP114_DT_ODR_6_4_MS
+      - 1    # TMP114_DT_ODR_32_25_MS
+      - 2    # TMP114_DT_ODR_62_5_MS
+      - 3    # TMP114_DT_ODR_125_MS
+      - 4    # TMP114_DT_ODR_250_MS
+      - 5    # TMP114_DT_ODR_500_MS
+      - 6    # TMP114_DT_ODR_1000_MS
+      - 7    # TMP114_DT_ODR_2000_MS
```

Comment:
Move the enum descriptions into the description property so they get rendered in the documentation
https://docs.zephyrproject.org/latest/build/dts/bindings-upstream.html#dt-bindings-default-rules